### PR TITLE
Fix disabling append

### PR DESCRIPTION
**Pull request by toolmantim**
Allows append to be disabled, and also makes some of the labels a little clearer around create/update.

annotate-1

annotate-2

### CODE REVIEW

File: .buildkite/pipeline.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 steps:
-  - block: 📝 Annotate
+  - block: "📝 ${ANNOTATE_LABEL:-Annotate}"
```
Comment (ticky) [inline .buildkite/pipeline.yml:2]: oh, fancy!

File: .buildkite/pipeline.yml [hunk 2]
```diff
@@ -44,6 +44,16 @@ steps:
...
 
+      - select: Append
+        hint: "Append to the body of an existing annotation?"
+        key: append
+        default: append
+        options:
+          - label: 'Yes'
+            value: append
+          - label: 'No'
+            value: no-append
```
Comment (ticky) [inline .buildkite/pipeline.yml:55]: that’s a much nicer solution, haha

File: .buildkite/pipeline.yml [hunk 3]
```diff
@@ -59,15 +69,6 @@ steps:
...
-        options:
-          - label: 'No'
-            value: ' '
-          - label: 'Yes'
-            value: '--append'
-
   - label: 👀
     command: |
-      buildkite-agent meta-data get body | buildkite-agent annotate --style "$(buildkite-agent meta-data get style)" $(buildkite-agent meta-data get append); buildkite-agent pipeline upload
+      buildkite-agent meta-data get body | buildkite-agent annotate --style "$(buildkite-agent meta-data get style)" $([[ $(buildkite-agent meta-data get append) == 'append' ]] && echo '--append'); env ANNOTATE_LABEL="Update" buildkite-agent pipeline upload
```
Comment (ticky) [inline .buildkite/pipeline.yml:74]: Is this getting complex enough to warrant moving it to a script? 🤔

**Review by ticky — APPROVED** [review]
Looks good!

File: .buildkite/pipeline.yml (hunk 1 above)
Comment (toolmantim) [reply .buildkite/pipeline.yml:2]: 😁

File: .buildkite/pipeline.yml (hunk 3 above)
Comment (toolmantim) [reply .buildkite/pipeline.yml:74]: Nar! I reckon it's a cool one-liner demo.

**Comment by toolmantim** [comment]
I think the webhooks are still hooked up to your own BK org @ticky

**Comment by ticky** [comment]
They are 😃 want to set it up under the Buildkite namespace?

**Pull request merged**